hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -24,199 +24,356 @@
+            Perform accesses in order to provoke error out of bound value. Test both, Software
+            interface and Hardware interface. Related covergroup is error_cg.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: lc_ctrl_if
+      name: error_mp
       desc: '''
-            TBD - verify the interface between flash controller and the life cycle controller.
+            Perform accesses in order to provoke memory permission error. Test both, Software
+            interface and Hardware interface. Related covergroup is error_cg.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: key_mgr_if
+      name: error_rd
       desc: '''
-            TBD - verify the seed output to the key manager.
+            Perform accesses in order to provoke read data error. Test both, Software interface and
+            Hardware interface. Related covergroup is error_cg.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: scramble
+      name: error_prog_win
       desc: '''
-            Enable scrambling, along with randomized scramble keys. Program a fresh chunk of
-            memory and read back (both, via controller and host) to ensure data integrity. On
-            program, verify via backdoor scrambling was done on the raw data correctly. When
-            reading via host, read the same memory via host multiple times back-to-back and ensure
-            the timing is correct (subsequent reads should be faster). When scrambling is not
-            enabled, ensure that the raw data is written and read back.
+            Perform accesses in order to provoke program resolution error. Test both, Software
+            interface and Hardware interface. Related covergroup is error_cg.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: ecc
+      name: error_prog_type
       desc: '''
-            Randomly enable ECC for a randomly selected set of pages. Randomly corrupt a single bit
-            in the memory that is about to be read and ensure that the ECC works - the corrupted bit
-            should be fixed. Corrupt randomly either the data or the ECC bits. Randomly corrupt 2
-            bits in the same word and ensure that the read results in error. Ensure that pages with
-            ECC not enabled reads back corrupted data without any errors.
+            Perform accesses in order to provoke program type error. Test both, Software interface
+            and Hardware interface. Related covergroup is error_cg.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: error_flash_phy
+      desc: '''
+            Perform accesses in order to provoke native flash error. Test both, Software interface
+            and Hardware interface. Related covergroup is error_cg.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: error_shadow_update
```

Comment:
remove this and import `shadow_reg_errors_testplan.hjson` for shadow_update and error_storage.

Reply:
removed error_integrity, shadow_update and error_storage tests. Removed also from CG. Added shadow_reg_errors_testplan.hjson